BoxContentSDK/BoxContentSDK/Requests/BOXCollectionItemsRequest.m

```diff
@@ -6,6 +6,7 @@
 #import "BOXRequest_Private.h"
 #import "BOXCollectionItemsRequest.h"
 #import "BOXAPIJSONOperation.h"
+#import "BOXDispatchHelper.h"
```

Comment:
This is because it used to get brought it implicitly from the .h of the super. Now that it's being imported explicitly only when needed, everything else continues to work as-is.